index.bs

```diff
@@ -2505,44 +2527,56 @@ optionally evidence of [=user consent=] to a specific transaction.
+        {{PublicKeyCredentialUserEntity/id}} can be returned as the {{AuthenticatorAssertionResponse/userHandle}}
+        in some future [=authentication ceremonies=],
+        and is used to overwrite existing [=discoverable credentials=]
+        with the same {{PublicKeyCredentialUserEntity/id}} on the same [=authenticator=].
+        {{PublicKeyCredentialEntity/name}} and {{PublicKeyCredentialUserEntity/displayName}}
+        MAY be used by the [=authenticator=] and [=client=] in future [=authentication ceremonies=]
+        to help the user select a [=credential=], but are not returned to the [=[RP]=].
+
+        For further details, see [[#dictionary-pkcredentialentity]] and [[#dictionary-user-credential-params]].
 
     :   <dfn>challenge</dfn>
-    ::  This member contains a challenge intended to be used for generating the newly created credential's [=attestation
+    ::  This member contains a challenge to be used for generating the newly created credential's [=attestation
         object=]. See the [[#sctn-cryptographic-challenges]] security consideration.
 
     :   <dfn>pubKeyCredParams</dfn>
-    ::  This member contains information about the desired properties of the credential to be created. The sequence is ordered
-        from most preferred to least preferred. The [=client=] makes a best-effort to create the most preferred credential that it
-        can.
+    ::  This member lists the key types and signature algorithms the [=[RP]=] supports,
+        ordered from most preferred to least preferred.
+        The [=client=] and [=authenticator=] make a best-effort to create a credential of the most preferred type possible.
+        If none of the listed types can be created, the [=registration ceremony=] fails.
 
     :   <dfn>timeout</dfn>
-    ::  This member specifies a time, in milliseconds, that the caller is willing to wait for the call to complete. This is
+    ::  This OPTIONAL member specifies a time, in milliseconds, that the [=[RP]=] is willing to wait for the call to complete. This is
         treated as a hint, and MAY be overridden by the [=client=].
 
     :   <dfn>excludeCredentials</dfn>
-    ::  This member is intended for use by [=[RPS]=] that wish to limit the creation of multiple credentials for the same
-        account on a single authenticator. The [=client=] is requested to return an error if the new credential would be created
-        on an authenticator that also contains one of the credentials enumerated in this parameter.
+    ::  This OPTIONAL member lists [=credentials=] the user has already registered,
+        and is used to ensure that the new credential is [=created on=] an [=authenticator=] not already registered.
+        The [=client=] and [=authenticators=] will prevent the user from creating the new credential on an authenticator
+        that also [=contains=] one of the credentials listed in {{PublicKeyCredentialCreationOptions/excludeCredentials}}.
```

Comment:
This was [addressed in #1649](https://github.com/w3c/webauthn/pull/1649/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985R2572) which is now merged in here. I dropped a few words that seemed redundant (like "on this RP" since a user account is by definition bound to a particular RP).